module/initComment.js

```diff
@@ -0,0 +1,13 @@
+const comments = async () => fetch(
+  'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/',
+  {
+    method: 'POST',
+  },
+)
+  .then((response) => response.text())
+  .then((data) => {
+    const postId = data.trim();
+    return postId;
+  });
+
+export default comments;
```

Comment:
- [ ] Please consider using the ES6 `async` and `await` instead of the `.then.catch` chains as one of the goals of this project is that you get used to the newer syntax! 🤓 🔧